### ISSUE TITLE
Calibrator check not correct.

### ISSUE THREAD

**Issue by PeterKamphuis** [caracal-pipeline/caracal#1115]
After my flagging woes I get the nice error:

```
2020-05-18 13:36:15 CARACal ERROR: The flux calibrator field "3C286" could not be found in our database or in the CASA NRAO database [RuntimeError]
```
However, this has never before been an issue and clearly 3C286 is part of the casa flux calibrator database. See here:

https://casa.nrao.edu/casadocs/casa-5.1.0/reference-material/flux-calibrator-models

So the check on the casa database is not correct somehow.

**Comment by KshitijT** [comment]
What's the field name specified as in the MS?

**Comment by PeterKamphuis** [comment]
@KshitijT 3C286

**Comment by KshitijT** [comment]
Could you post your crosscal settings?

**Comment by PeterKamphuis** [comment]
```
crosscal:
  enable: true
  uvrange: '>1klambda'
  label_in: 'cal'
  set_model:
    enable: true
  primary:
    reuse_existing_gains: true
    order: GBAGB
    combine: [ "", "", null, "",""]
    solint: ['inf', 'inf,3ch', null, 'inf', 'inf']
    calmode: [p, ap, null, ap, ap] 
    b_fillgaps: 70
    plotgains: true
  secondary:
    reuse_existing_gains: False
    order: GBAGB
    apply: B
    combine: ["","", null,"",""]
    solint: ['32s', "inf,3ch",null,"32s","inf,3ch"]
    calmode: [ap,ap,null,ap,ap]
    plotgains: True
  apply_cal: 
    applyto: 
      - gcal
      - bpcal
```

**Comment by PeterKamphuis** [comment]
@KshitijT Please note that this is a set without intents and the calibrators are set explicitly in the yml file.

```
obsconf:
  refant: '10'
  target: ['2014AD']
  fcal: ['3C147']
  bpcal: ['3C147']
  gcal: ['1130-148']
```
Testing 3C147 at the moment but it was exactly the same for 3C286.

**Comment by PeterKamphuis** [comment]
@KshitijT 3C147 works so I don't know what is going on.

**Comment by PeterKamphuis** [comment]
Ok this is going wrong in the location matching but I'm not sure why we care about that if we have a name?

**Comment by KshitijT** [comment]
Yeah, I was trying to dig into that. But I think it's much more easier to do this:

**Comment by PeterKamphuis** [comment]
They are far off though?
```
Checking these coord
3C286 [3.5392586514514845, 0.5324854103730366] [-2.743927253136695, 0.5324851588008706]
```
Maybe the 3c286 coordinates are wrong in the database?

**Comment by KshitijT** [comment]
What are far off?

I was looking at this: https://github.com/caracal-pipeline/caracal/blob/master/caracal/data/casa_calibrators.txt#L9

`name=3C286 epoch=0000 ra=13h31m08.3s dec=+30d30m33s a=0.0 b=0.0 c=0.0 d=0.0` .

**Comment by PeterKamphuis** [comment]
Maybe the - is the issue? Those are checked in radians right?

**Comment by PeterKamphuis** [comment]
from the .txt obsconf

```
  1         3C286               13:31:08.291785 +30.30.32.94811 J2000   1          20010
```
So that is right.

**Comment by PeterKamphuis** [comment]
yeah the issue is the minus, 2pi-2.743927253136695 = 3.53926

**Comment by gigjozsa** [comment]
@PeterKamphuis I cannot follow. Where in the code?

**Comment by PeterKamphuis** [comment]
just a sec I'll make a PR with a fix and request your review

**Comment by gigjozsa** [comment]
Tnx!

**Comment by PeterKamphuis** [comment]
@gigjozsa Please see PR #1119. The issue was that for 3C286 apparently this data set had a negative direction in the `delay-dir` .

**Comment by gigjozsa** [comment]
@PeterKamphuis Yes, I just saw that (and I made a suggestion)

**Comment by PeterKamphuis** [comment]
Fixed in PR #1119
